roles/bootstrap-os/tasks/main.yml

```diff
@@ -6,97 +6,50 @@
   # This command should always run, even in check mode
   check_mode: false
 
-- name: Bootstrap CentOS
-  include_tasks: bootstrap-centos.yml
-  when: '''ID="centos"'' in os_release.stdout_lines or ''ID="ol"'' in os_release.stdout_lines or ''ID="almalinux"'' in os_release.stdout_lines or ''ID="rocky"'' in os_release.stdout_lines or ''ID="kylin"'' in os_release.stdout_lines  or ''ID="uos"'' in os_release.stdout_lines or ''ID="openEuler"'' in os_release.stdout_lines'
+- name: Include distro specifics vars and tasks
+  vars:
+    os_release_dict: "{{ os_release.stdout_lines | select('regex', '^.+=.*$') | map('regex_replace', '\"', '') |
+                         map('split', '=') | community.general.dict }}"
+  block:
+  - name: Include vars
+    include_vars: "{{ item }}"
+    tags:
+    - facts
+    with_first_found:
+    - &search
+      files:
+      - "{{ os_release_dict['ID'] }}-{{ os_release_dict['VARIANT_ID'] }}.yml"
+      - "{{ os_release_dict['ID'] }}.yml"
```

Comment:
It's a bit a nit but I kinda like the fact there was a prefix "bootstrap-" there but I am fine with that if you prefer this.

Reply:
I tend to dislike repeated prefix stuff when there is already a "namespace" (roles/**boostrap**-os/tasks/**bootstrap**-*), it tends to make me cringe :smile: . But that's something really important, I'm fine with either way.

Reply:
Hmm well to me that was more something to quickly distinguish the code that loads thought this than the rest but as it's only main.yml it's not super important either